cl/opinion_page/tests.py

```diff
@@ -210,3 +218,132 @@ def test_does_the_sitemap_have_content(self):
         # Class attributes are set, just run the test in super.
         self.expected_item_count = self.get_expected_item_count()
         super(OpinionSitemapTest, self).does_the_sitemap_have_content()
+
+
+class UploadPublication(TestCase):
+
+    fixtures = ["tenn_judges.json"]
+
+    def setUp(self):
+        self.client = Client()
+        tenn_group = Group.objects.get(name="tenn_work_uploaders")
+        self.tenn_user = User.objects.create_user(
+            "learned", "learnedhand@scotus.gov", "thehandofjustice"
+        )
+        self.reg_user = User.objects.create_user(
+            "test_user", "test_user@scotus.gov", "simplepassword"
+        )
+
+        self.tenn_user.groups.add(tenn_group)
+        self.tenn_user.save()
```

Comment:
I don't think you need this line? I think `groups.add()` probably does the save to the DB.

Reply:
Not required.  Removed